ci-operator/step-registry/baremetalds/packet/check/baremetalds-packet-check-commands.sh

```diff
@@ -32,7 +32,7 @@ leak_num="$(echo "$leak_ids" | wc -w)"
 leak_report="${leak_report}\nProw job references per leaked server:" 
 for server in $leak_servers
 do
-  leak_report="${leak_report}\n<https://search.apps.build01.ci.devcluster.openshift.com/?search=$server&maxAge=48h&context=-1&type=build-log|$server>"
+  leak_report="${leak_report}\n<https://search.ci.openshift.org/?search=$server&maxAge=48h&context=-1&type=build-log|$server>"
```

Comment:
What about instead putting a direct link to the PR dashboard? Now that we've the tags all the elements to compose the URL should be available

Reply:
We can use PR link in case of pre-submit job and stick with the above search for periodics (PR tag is empty). What do you think?

Reply:
TBH I'd prefer to have a direct job dashboard link whenever a leak is detect and notified regardless of the original kind of job, so that whoever is in charge for the @ metal-support role could simply single-click on the link to access the job details.
If it's too problematic now it's fine to close this PR as a simple fix for the existing search URL.

Reply:
I added PR links, PTAL.